docs/usage.md

```diff
@@ -40,7 +40,7 @@ For CSV input files, specifying the `batch_col`, `label_col`, and `unknown_label
 | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `sample`          | Unique sample identifier. Will be added to the pipeline output objects as `sample` column.                                                                                                                                                                                                                                                                                                                                               |
 | `file`            | May contain paths to `h5ad`, `h5`, `rds`, or `csv` files. If used with cellranger or nf-core/scrnaseq, this should contain the filtered output. `rds` files may contain any object that can be converted to a `SingleCellExperiment` using the [Seurat `as.SingleCellExperiment`](https://satijalab.org/seurat/reference/as.singlecellexperiment) function. `csv` files should contain a matrix with genes as columns and cells as rows. |
```

Comment:
Technically an entry to the file column is always required, that aside, are the filtered files required when using cellbender for ambient processing?

Reply:
The wording could be better here. The file column must be defined, but it may be any out of `h5ad`, `h5`, `rds` or `csv`.
Cellbender is meant to be only used when coming from cellranger (which is also used inside of nf-core/scrnaseq). In this case, the `file` column should contain the filtered output.

Reply:
I believe cellbender could also make sense for other outputs from nf-core/scrnaseq. See https://nf-co.re/scrnaseq/2.7.0/docs/output/#other-output-data, which states:

> Some aligners, like alevin do not produce both raw&filtered matrices. When aligners give only one output, they are treated with the raw suffix. Some aligners may have an option to give both raw&filtered and only one, like kallisto. Be aware when using the tools.

Cellbender is not very explicit about other tools than cellranger, but they mention kallisto in the code (https://github.com/broadinstitute/CellBender/blob/04c2f5b460721fd55cf62a4cd23617b2555d69b8/cellbender/remove_background/data/io.py#L1034)

Reply:
What I wrote [here](https://github.com/nf-core/scdownstream/issues/51#issuecomment-2205916932) is related to this topic, and probably the better place to discuss this is in #51